### PR TITLE
Test case test_worker_node_restart_during_pvc_expansion enhancement

### LINKED ISSUE #2750: Test case improvement - test_worker_node_restart_during_pvc_expansion
Suggesting few improvements in test case ocs-ci/tests/manage/pv_services/pvc_resize/test_node_restart_during_pvc_expansion.py

1. Check new size on pod after volume expansion.
2. Add describe output of PVC and PV if expansion fails.

### PR DESCRIPTION
**Pull request by jilju**
Test case test_worker_node_restart_during_pvc_expansion enhancement.
    
1. Check new size on pod after volume expansion.
2. Add describe output of PVC and PV if expansion fails.

Closes #2750 

Signed-off-by: Jilju Joy <jijoy@redhat.com>

### CODE REVIEW

File: tests/manage/pv_services/pvc_resize/test_node_restart_during_pvc_expansion.py [hunk 1]
```diff
@@ -86,10 +86,51 @@ def test_worker_node_restart_during_pvc_expansion(self, nodes):
...
+                if new_size_mount in [
+                    f'{pvc_size_expanded - 0.1}G',
+                    f'{float(pvc_size_expanded)}G',
+                    f'{pvc_size_expanded}G'
+                ]:
+                    log.info(
+                        f"Verified: Expanded size of PVC {pod_obj.pvc.name} "
+                        f"is reflected on pod {pod_obj.name}"
+                    )
+                    break
```
Comment (vavuthu) [inline tests/manage/pv_services/pvc_resize/test_node_restart_during_pvc_expansion.py:122]: Do you want to add assert if new_size_mount is not in the pvc_size-expanded list ?

Comment (jilju) [reply tests/manage/pv_services/pvc_resize/test_node_restart_during_pvc_expansion.py:122]: Retry will be done. If it not reflected within the specified time, TimeOutExpired error will be raised.

**Review by sidhant-agrawal — DISMISSED** [review]

File: tests/manage/pv_services/pvc_resize/test_node_restart_during_pvc_expansion.py [hunk 2]
```diff
@@ -92,11 +92,49 @@ def test_worker_node_restart_during_pvc_expansion(self, nodes):
...
+            if pod_obj.pvc.volume_mode == "Block":
+                log.info(
+                    f"Skipping check on pod {pod_obj.name} as volume mode is Block."
+                )
+                continue
+
+            # Wait for 240 seconds to reflect the change on pod
+            log.info(f"Checking pod {pod_obj.name} to verify the change.")
+            for df_out in TimeoutSampler(
+                240, 3, pod_obj.exec_cmd_on_pod, command="df -kh"
```
Comment (vasukulkarni) [inline tests/manage/pv_services/pvc_resize/test_node_restart_during_pvc_expansion.py:113]: worth moving to utils so that others can utilize

Comment (jilju) [reply tests/manage/pv_services/pvc_resize/test_node_restart_during_pvc_expansion.py:113]: Good to add a helper function which can fetch the total size, used size or used percentage as needed.

**Review by vavuthu — APPROVED** [review]

**Review by sidhant-agrawal — APPROVED** [review]

**Pull request merged**